crates/diesel_models/src/payment_method.rs

```diff
@@ -142,6 +280,63 @@ impl PaymentMethodUpdate {
     }
 }
 
+#[cfg(all(feature = "v2", feature = "payment_methods_v2"))]
+#[derive(
+    Clone, Debug, Default, AsChangeset, router_derive::DebugAsDisplay, Serialize, Deserialize,
+)]
+#[diesel(table_name = payment_methods)]
+pub struct PaymentMethodUpdateInternal {
```

Comment:
Can we have `last_modified` as a required field in both the `PaymentMethodUpdateInternal` structs, so that we can ensure that the field is always updated with any update to the payment method record? (You would have to remove the `Default` derive on `PaymentMethodUpdateInternal` as a result.)